Sources/Rendering/Misc/TextureLODsDownloader/index.js

```diff
@@ -0,0 +1,105 @@
+import macro from 'vtk.js/Sources/macro';
+
+const { vtkErrorMacro } = macro;
+
+// ----------------------------------------------------------------------------
+// vtkTextureLODsDownloader methods
+// ----------------------------------------------------------------------------
+
+function vtkTextureLODsDownloader(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkTextureLODsDownloader');
+
+  const internal = {
+    downloadStack: [],
+  };
+
+  //--------------------------------------------------------------------------
+
+  publicAPI.startDownloads = () => {
+    if (!model.texture) {
+      vtkErrorMacro('Texture was not set.');
+      return;
+    }
+
+    if (!model.files || model.files.length === 0) {
+      vtkErrorMacro('No files set.');
+      return;
+    }
+
+    let baseUrl = model.baseUrl;
+    if (baseUrl && !baseUrl.endsWith('/')) {
+      baseUrl += '/';
+    }
+
+    // Create the download stack
+    internal.downloadStack = [];
+    model.files.forEach((file) => internal.downloadStack.push(baseUrl + file));
```

Comment:
You probably want to use a template string. I would have thought the linter would warn you on it.

Reply:
I changed this to use a template string.